claat/parser/gdoc/parse.go

```diff
@@ -650,9 +650,27 @@ func list(ds *docState) types.Node {
 // image creates a new ImageNode out of hn, parsing its src attribute.
 // It returns nil if src is empty.
 // It may also return a YouTubeNode if alt property contains specific substring.
+// or an IframeNode if the alt property contains a URL other than youtube.
 func image(ds *docState) types.Node {
-	if strings.Contains(nodeAttr(ds.cur, "alt"), "youtube.com/watch") {
+	alt := nodeAttr(ds.cur, "alt")
+	if strings.Contains(alt, "youtube.com/watch") {
 		return youtube(ds)
+	} else if strings.Contains(alt, "https://") {
```

Comment:
Could we do "Starts With" rather than "Contains" otherwise a non-https URL that has "https://" in the URL (as a query param for instance) will be accepted :P)

Reply:
Ah it's fine since you double check afterwards in `func iframe`